src/usr/share/batwatch/batwatch.py

```diff
@@ -53,124 +46,150 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
     def __ne__(self, other):
-        return not self.__eq__(other)
+        return not self == other
 
+    def __gt__(self, other):
+        is_greater_than = False
+        if self.battery_count > other.battery_count:
+            is_greater_than = True
 
-class Batwatch(object):
-    """
-    Monitors the status of system battery and notifies designated recipient(s) of changes
-    by gpgmailer encrypted email.
-    """
+        elif self.battery_count == other.battery_count:
+            if self.charge_status > other.charge_status:
+                is_greater_than = True
```

Comment:
I probably told you to do this but you really can't determine whether the log message should be info or warning with inequality alone. Technically, if either battery count or charge status goes down, a warning should be logged. Trying to capture this with inequality would make it so that both a < b and b < a is true at the same time. Obviously that is bad design.

You might say that the chance of the battery count and charge status changing in the same loop is minuscule, but consider the case of someone plugging in an unplugged UPS.